doc/teehistorian.md

```diff
@@ -99,6 +99,12 @@ The following extra messages are known right now:
 * TEEHISTORIAN_TEAM_PRACTICE(teehistorian-team-practice@ddnet.tw): team(int), practice(int) records when a team enters practice mode, resulting ranks don't get submitted to the database
   * uuid: 5792834e-81d1-34c9-a29b-b5ff25dac3bc
   * introduced in DDNet 15.6, [81f4263428069526](https://github.com/ddnet/ddnet/commit/81f426342806952603a2d28290279e0a7107db5b)
+* TEEHISTORIAN_PLAYER_READY(teehistorian-player-ready@ddnet.tw): cid(int) records client message that it is ready to join the game
```

Comment:
It doesn't actually record the client message, does it? Perhaps "records when the client messages that it is ready to join the game", but even better would be a description of what that means, i.e. probably that the server starts spawning the client in the gameworld etc.